SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -37,11 +50,15 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 			: SpatialOSDirectory.Path;
 	}
 
-	FORCEINLINE FString GetGeneratedSchemaOutputFolder() const
+	FORCEINLINE FString GetSpatialOSLaunchConfig() const
 	{
-		return GeneratedSchemaOutputFolder.Path.IsEmpty()
-			? FPaths::ConvertRelativePathToFull(FPaths::Combine(GetSpatialOSDirectory(), FString(TEXT("schema/unreal/generated/"))))
-			: GeneratedSchemaOutputFolder.Path;
+		return SpatialOSLaunchConfig.IsEmpty() ? FString(TEXT("default_launch.json")) : SpatialOSLaunchConfig;
+	}
+
+	UFUNCTION()
```

Comment:
Why was this made a UFUNCTION()?

Reply:
Original definition in SpatialGDKEditorToolbarSettings.h (deleted in this PR) had defined it as a UFUNCTION, probably save to delete.